tests/test_misc.py

```diff
@@ -794,6 +796,27 @@ def fn(x):
         self.assertEqual(cnts.frame_count, 1)
         self.assertEqual(cnts.op_count, 9)
 
+    def test_store_global_1(self):
+        class MyObj:
+            def __init__(self, a):
+                self.a = a
+
+        def fn(obj):
+            global _gtensor
+            val = obj.a + _gtensor
+            _gtensor += torch.ones(10)
```

Comment:
I'd suggest a few more test cases:
1) writing to a new global: `global _new_global; _new_global = obj.a+1;`
2) writing to a global defined in a different file while inlining a function (this will be harder because we cant use `STORE_GLOBAL` in output code.)